WordPressKit/RemoteSourcePostAttribution.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+@objcMembers public class RemoteSourcePostAttribution: NSObject {
+
+    public var permalink: String?
+    public var authorName: String?
+    public var authorURL: String?
+    public var blogName: String?
+    public var blogURL: String?
+    public var avatarURL: String?
+    public var blogID: NSNumber?
+    public var postID: NSNumber?
+    public var likeCount: NSNumber?
+    public var commentCount: NSNumber?
+    public var taxonomies: NSArray?
```

Comment:
Does Swift know about the underlying type here? If so, can we use a Swift collection?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPressKit-iOS/pull/560/commits/d2d11251072ac48a8df2c5fe09a05b54b5a8980b